roles/ces_common/tasks/configure.yml

```diff
@@ -42,8 +42,8 @@
 
 - name: configure | Collect all nodes on which ces is not enabled
   set_fact:
-    scale_ces_disabled_nodes: "{{ scale_ces_disabled_nodes + [ item ]}}"
-  when: not scale_ces_enable_status.stdout_lines is search(item)
+    scale_ces_disabled_nodes: "{{ scale_ces_disabled_nodes + [hostvars[item]['scale_daemon_nodename']] }}"
```

Comment:
@acch Can we keep inventory_hostname instead of scale_daemon_nodename ?

Reply:
We'll use these names with `mmchnode -N`, which will require the nodename as opposed to the inventory name.